verify_sri.py

```
@@ -0,0 +1,22 @@
+import base64
+
+urls = [
+    "https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism-tomorrow.min.css",
+    "https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/plugins/line-numbers/prism-line-numbers.min.css",
+    "https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/plugins/toolbar/prism-toolbar.min.css",
+    "https://cdn.tailwindcss.com/3.4.1"
+]
+
+for url in urls:
+    try:
+        response = requests.get(url)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)